### DISCUSSION TITLE
[FEATURE]: getListingsByShopSectionId with state query parameter

### DISCUSSION THREAD

**Discussion by furnaceX** [etsy/open-api#728] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

While getting listing from the getListingsByShopSectionId endpoint, there doesn't seem to be a way to get expired, inactive, etc. listing.

### Describe the solution you'd like

add "state" as a query parameter, similar to what is in the getListingsByShop endpoint.

**Comment by etsyachristensen** [comment]
@furnaceX I'm moving this to a new Idea discussion because we'll be closing the issues board in the near future
